CMakeLists.txt

```diff
@@ -145,27 +145,6 @@ add_library (mod_odb STATIC ${MOD_ODB_SOURCES})
 target_link_libraries (mod_odb ${THIRD_PARTY_LIBRARIES})
 
 set (MOD_VCF_SOURCES
-  inc/vcf/debugulator.hpp
```

Comment:
this is not for including the libraries, it's for adding them in the dependency graph, so that when a change is detected in one of these files, all the objects that depend on them are recompiled. Please restore this part.